app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/informdeflect/DeflectCarOtherDamageDestination.kt

```diff
@@ -69,6 +76,17 @@ private fun DeflectCarOtherDamageScreen(
     closeClaimFlow = closeClaimFlow,
     topAppBarText = stringResource(id = R.string.SUBMIT_CLAIM_CAR_TITLE),
   ) {
+    var shouldCloseFlow by remember {
+      mutableStateOf(false)
+    }
+
+    LaunchedEffect(shouldCloseFlow) {
+      if (shouldCloseFlow) {
+        delay(3.minutes) // todo: on the test session decided to make a delay here
```

Comment:
We do not get any guarantees that our app will be kept running in the background for this long in order to do this reliably. If the app is killed inside those 3 minutes this will simply never be called. I will bring this up in the slack channel instead to pick alternatives. I would prefer us not to do this and get unreliable results.